test/test_other.py

```diff
@@ -15804,3 +15804,6 @@ def test_rollup(self):
 
   def test_rlimit(self):
     self.do_other_test('test_rlimit.c', emcc_args=['-O1'])
+
+  def test_stub_pthread_getattr_np(self):
+    self.do_runf(test_file('other/test_stub_pthread_getattr_np.c'), 'stack_addr: 0, stack_size: 65536, guard_size: 8192')
```

Comment:
No need for `test_file` here.

Reply:
Maybe use `.out` file alongside the `.c` file?

Then you would use `do_other_test`... and you can run with `--rebase` to generate the .out content.

Reply:
LGTM either way though.

Reply:
`--rebase` is convenient.